### PR TITLE
Updating TeamsPage Tables

### PR DESCRIPTION
**Pull request by katherinj**
Updated TeamsPage and ProjectsPage tables to use Material Table library for a better UI. In doing so, we realized we would need an additional route/ function in the teams model and router to be able to get an array of user objects that a team belongs to. This route and function has been added and a function called "fetchMemberList" has been added to the apiClient. Minor styling fixes have been made on both the TeamsPage and ProjectsPage.

### CODE REVIEW

File: bugtracker-ui/src/components/Tables/ProjectsPageTicketsTable.jsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import { useNavigate } from "react-router-dom";
+
+/**
+ * Gets called when a row is clicked.
+ * Takes the data of the row and console logs it.
+ * You can use this data to pull id or anything else you want,
+ * and use it for other purposes.
+ */
+function onRowClick(data) {
+  console.log("Row data below!");
```
Comment (LifeBringer) [inline bugtracker-ui/src/components/Tables/ProjectsPageTicketsTable.jsx:18]: Remove/delete

File: bugtracker-ui/src/components/Tables/ProjectsPageTicketsTable.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  console.log(data);
```
Comment (LifeBringer) [inline bugtracker-ui/src/components/Tables/ProjectsPageTicketsTable.jsx:19]: Remove/delete

File: bugtracker-ui/src/components/Tables/TeamsPageDevelopersTable.jsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+ *
+ *
+ */
+
+import MaterialTable from "material-table";
+import { useNavigate } from "react-router-dom";
+
+// IGNORE THIS
+const handleOnRowClick = (rowData) => {
+  console.log("Clicked on row!");
```
Comment (LifeBringer) [inline bugtracker-ui/src/components/Tables/TeamsPageDevelopersTable.jsx:12]: Remove/delete

File: bugtracker-ui/src/components/Tables/TeamsPageDevelopersTable.jsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  console.log(rowData.id);
```
Comment (LifeBringer) [inline bugtracker-ui/src/components/Tables/TeamsPageDevelopersTable.jsx:13]: Remove/delete

File: bugtracker-ui/src/components/Tables/ProjectsPageTicketsTable.jsx (hunk 2 above)
Comment (Rebecca-CYLi) [inline bugtracker-ui/src/components/Tables/ProjectsPageTicketsTable.jsx:19]: console log?

**Review by LifeBringer — CHANGES_REQUESTED** [review]
12 console logs, please resolve.

File: bugtracker-ui/src/components/Tables/TeamsPageDevelopersTable.jsx (hunk 4 above)
Comment (Rebecca-CYLi) [inline bugtracker-ui/src/components/Tables/TeamsPageDevelopersTable.jsx:13]: console log?

File: bugtracker-ui/src/components/Tables/TeamsPageDevelopersTable.jsx [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+/**
+ * Gets called when a row is clicked.
+ * Takes the data of the row and console logs it.
+ * You can use this data to pull id or anything else you want,
+ * and use it for other purposes.
+ */
+function onRowClick(data) {
+  console.log("Row data below!");
+  console.log(data);
```
Comment (Rebecca-CYLi) [inline bugtracker-ui/src/components/Tables/TeamsPageDevelopersTable.jsx:24]: console log?

File: bugtracker-ui/src/components/Tables/TeamsPageProjectsTable.jsx [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+ */
+
+import MaterialTable from "material-table";
+import { useNavigate } from "react-router-dom";
+
+// IGNORE THIS
+const handleOnRowClick = (rowData) => {
+  console.log("Clicked on row!");
+  console.log(rowData.id);
+};
```
Comment (Rebecca-CYLi) [inline bugtracker-ui/src/components/Tables/TeamsPageProjectsTable.jsx:14]: console log?

File: bugtracker-ui/src/components/Tables/TeamsPageProjectsTable.jsx [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+/**
+ * Gets called when a row is clicked.
+ * Takes the data of the row and console logs it.
+ * You can use this data to pull id or anything else you want,
+ * and use it for other purposes.
+ */
+function onRowClick(data) {
+  console.log("Row data below!");
+  console.log(data);
```
Comment (Rebecca-CYLi) [inline bugtracker-ui/src/components/Tables/TeamsPageProjectsTable.jsx:24]: console log?

**Review by Rebecca-CYLi — CHANGES_REQUESTED** [review]
some console logs to remove

**Review by LifeBringer — APPROVED** [review]

**Pull request merged**